src/Image.php

```diff
@@ -409,32 +422,42 @@
  */
 class Image extends ImageAutodoc implements \ArrayAccess
 {
+
     /**
-     * Set true when we are logging actions.
+     * The logger instance.
      *
-     * @internal 
+     * @var LoggerInterface
      */
-    static private $_enable_logging = false;
+    private static $logger;
```

Comment:
Looks like the autogenerated code needs some work to pass phpcs PSR2 as well. Something else for the TODO. 

Or is phplint the preferred tool for this?